server/googleActions.js

```diff
@@ -136,7 +134,7 @@ const GoogleActions = {
 	},
 
 	updateGoogleBrew : async (brew)=>{
```

Comment:
Is there a specific reason we're not passing the OAuth2 client in the params as `auth` as we're doing with the other functions? e.g. `newGoogleBrew(auth, brew)`, `listGoogleBrews (auth)`

Reply:
Yes, there is a reason.  At the top of this file we already set the `serviceAuth` which makes it the default for all of the api requests (line 25 above which you deleted). The only cases where we pass in a separate `auth` is when we want to require the specific user to be logged in.

For example, anybody with editor access to the file can update it (uses `serviceAuth`, no need to pass in `auth`). However only the document owner themselves is allowed to delete the file (we validate their user credentials and pass in that OAuth2 token instead).

So this change: 

```
const drive = drive({ version: 'v3', auth: serviceAuth || config.get('google_api_key') });
```

Should be undone, since the default was already being set with

```
google.options({ auth: serviceAuth || config.get('google_api_key') });
```

Reply:
Well, the global `google.options` is no longer available when using the Google Drive specific package. So I had to remove that line.

I believe the only way around this is by passing the auth to the `drive()` constructor as an option, which I've done in the auth commit.

Reply:
I see. In that case, let's just put an object up top to hold our default settings, and then pass in that object into the `drive()` constructor where needed. Just so all of our default options are handled in one place.

Something like:

```
defaultAuth = {serviceAuth || config.get('google_api_key')};


//
//
//

const drive = drive({ version: 'v3', auth: ...defaultAuth });

Reply:
@calculuschild Implemented your suggestion 👌